saleor/graphql/product/mutations/channels.py

```diff
@@ -137,9 +150,11 @@ def validate_product_without_category(cls, cleaned_input, errors: ErrorType):
             )
 
     @classmethod
-    def add_channels(cls, product: "ProductModel", add_channels: List[Dict]):
+    def add_channels(cls, info, product: "ProductModel", add_channels: List[Dict]):
```

Comment:
The name of this method shouldn't be changed to `update_channels`, as the input field name has been changed?